[Getting Started Docs][A/B test] Split docs (#6408)
PostHog/posthog.com: contents/docs/experiments/testing-and-launching.mdx

---
title: Testing and launching an experiment
sidebar: Docs
showTitle: true
availability:
    free: none
    selfServe: full
    enterprise: full
---

Once you've written your code, it's a good idea to test that each variant behaves as you'd expect. If you find out your implementation had a bug after you've launched the experiment, you lose days of effort as the experiment results can no longer be trusted.

> The best way to do this is adding an optional override to your release conditions. For example, you can create an override to assign a user to the 'test' variant if their email is your own (or someone in your engineering team). You can do this by:

Reviewer: ```suggestion
The best way to do this is adding an optional override to your release conditions. For example, you can create an override to assign a user to the 'test' variant if their email is your own (or someone in your engineering team). To do this:
```

1. Go to your experiment feature flag.
2. Ensure the feature flag is enabled by checking the "Enable feature flag" box.
3. Add a new condition set with the condition to `email = your_email@domain.com`. Set the rollout percentage for this set to 100%.
4. Set the optional override for the variant you'd like to assign these users to.
5. Click "Save".

![How to test your experiment](../../images/features/experiments/testing-your-experiment.mp4)

> Once you're satisfied, you're ready to launch your experiment.

Reviewer: ```suggestion
Once you test it works, you're ready to launch your experiment.
```

> > **Note:** The feature flag is activated only when you launch the experiment, or if you've manually checked the "Enable feature flag" box.

Reviewer: ```suggestion
```
Related to Andy's comment, I don't think you need this one.

> > **Note:** While the [PostHog toolbar](/docs/toolbar) enables you to toggle feature flags on and off, this only works for active feature flags and won't work for your experiment feature flag while it is still in draft mode.

Reviewer 2: Seems a bit weird to have two boxouts here. Is the toolbar one really necessary. could it go somewhere else/

Author: Perhaps not. I ran into this issue while testing once, so worht calling out
